pdns/recursordist/Makefile.am

```diff
@@ -368,6 +388,14 @@ pdns_recursor_SOURCES += \
 
 endif
 
+if FSTRM
+pdns_recursor_SOURCES += \
+	dnstap.cc dnstap.hh fstrm_logger.cc fstrm_logger.hh dolog.hh
```

Comment:
I don't think we should use `dolog.hh` in the rec, perhaps we can do the same kind of ugly hack we currently have in `remote_logger.cc` with `#ifdef PDNS_CONFIG_ARGS` ?

Reply:
> `#ifdef PDNS_CONFIG_ARGS`

We have a `RECURSOR` variable as well

Reply:
Will fix in upcoming commit (using RECURSOR)